Connect.vb

```diff
@@ -144,5 +149,81 @@ Public Class Connect
             MessageBox.Show(ex.ToString())
         End Try
     End Sub
+    
+     ''' <summary>
+    ''' Host a window by UserControlHost.
+    ''' </summary>
+    ''' <param name="toolWindowCaption">
+    ''' String you need to put as a header on the window.
+    ''' </param>
+    ''' <param name="toolWindowGuid">
+    ''' This uniquely identified a particular window can be used to store 
+    ''' information of it like size, position, etc.
+    ''' </param>
+    ''' <param name="toolWindowUserControl">
+    ''' Windows to host.
+    ''' </param>
+    Private Function CreateToolWindow(ByVal toolWindowCaption As String, ByVal toolWindowGuid As String,
+      ByVal toolWindowUserControl As UserControl) As Window
+        Dim userControlObject As Object = Nothing
+        Dim userControlHost As UserControlHost
+        Dim _SummaryWindow As Window
+        Dim progId As String
+        ' IMPORTANT: ensure that you use the same ProgId value used in the ProgId attribute of the UserControlHost 

```

Comment:
Do not add such prefix in comments 'IMPORTANT'

Reply:
Edited.